CONTRIBUTING.md

```diff
@@ -19,21 +19,31 @@ To get started working on TestEZ, you'll need:
 * [Luacheck](https://github.com/mpeterv/luacheck) (`luarocks install luacheck`)
 * [LuaCov](https://keplerproject.github.io/luacov) (`luarocks install luacov`)
 
-Once you have all of these installed, you can run `lua bin/install-dependencies.lua` script to grab a couple additional local dependencies automatically.
+Make sure to clone the repository with submodules. You can do that to your existing repository with:
+
+```sh
+git submodule update --init
+```
 
 Finally, you can run all of TestEZ's tests with:
 
 ```sh
-lua spec.lua
+lua test/lemur.lua
 ```
 
 Or, to generate a LuaCov coverage report:
 
 ```sh
-lua -lluacov spec.lua
+lua -lluacov test/lemur.lua
 luacov
 ```
 
+If you're an engineer at Roblox, you can skip this setup and use Roblox-CLI. Make sure it's on your `PATH` and that you have a production Roblox Studio installation. You can then run:
+
+```sh
+./test/roblox-cli.sh
+```
+
```

Comment:
Should we add instructions on how to run these tests in Studio since that target is also supported in this change?

Reply:
That's a good idea! This guide is duplicated in a lot of repositories, I wish we had a better system for propagating fixes like that.